### PR TITLE
Fix function and delete unnecessary parameters

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/fnTy5yik/225-feature-coreunittransparencyreporting-v5

# Actions
Fix function and delete unnecessary parameters

### CODE REVIEW

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Components/Finances/ExpensesChart',
+  component: ExpensesChart,
+  decorators: [
+    (Story: Story) => (
+      <div>
+        <Story />
+      </div>
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx:14]: This decorator is not necessary here as it doesn't do anything

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          componentStyle: {
+            width: 266.5,
+          },
+          style: {
+            top: -1,
+            left: -40,
+          },
+        },
+      },
+    },
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx:89]: Please, note that the Figma urls are not of the Chart component

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          borderRadius: [0, 0, 6, 6],
+        },
+
+        name: 'Active Budget',
+        value: 16965122,
+      },
+    ] as ValuesDataWithBorder[],
+
+    newDiscontinued: [],
+    newPrediction: [],
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx:58]: You should add more data to see how the component will looks like in a real environment

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.tsx [hunk 4]
```diff
@@ -55,16 +52,12 @@ const ExpensesChart: React.FC<Props> = ({ monthly, newActual, newDiscontinued, n
...
-            return formatWithTwoTensAndDot + 'M';
-          } else if (value >= 1000) {
-            const formatWithTwoTensAndDot = replaceAllNumberLetOneBeforeDot(value, 10000);
-            return formatWithTwoTensAndDot + 'K';
-          } else {
+        formatter: function (value: number, index: number) {
+          if (value === 0 && index === 0) {
             return value.toString();
           }
+          console.log('value: ' + typeof value);
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.tsx]: not, i will remove it

**Pull request merged**